### PR TITLE
Pan4533

### PR DESCRIPTION
**Pull request by natashapinto**
This change is [Reviewable](https://reviewable.io/reviews/moka-guys/mokabed/238)

### CODE REVIEW

File: LiveBedfiles/RequestForms/Pan4566.csv [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+2. Panel description/name (fill in a-c from Moka),,,,,,,,,,
+2. a) Category,VCP3 Viapath,,,,,,,,,
+2. b) SubCategory,R58_Neurodegenerative disorder - Adult onset,,,,,,,,,
+2. c) Panel,VCP3 Viapath R58_Neurodegenerative disorder - Adult onset,,,,,,,,,
+3a). Is this BED file used for Exome depth?,YES,
+3b). Is this BED file used to generate read counts or filter calls?,filter calls,
+4. a) Is this BED file exclusively based on coordinates provided by a supplier ,No,,,,,,,,,
+4. b) If yes, does this need further padding?",N/A,,,,,,,,,
+5. a) Should specific transcripts be used to create the bed file? (as opposed to all transcripts) Specific transcripts - see Moka,Yes - see below,,,,,,,,,
+5. b) If specific transcripts are to be used have you provided multiple transcripts for the same gene? (if Yes please state),No,,,,,,,,,
```
Comment (aledj2) [inline LiveBedfiles/RequestForms/Pan4566.csv:16]: yes.

File: LiveBedfiles/RequestForms/Pan4533.csv [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+2. Panel description/name (fill in a-c from Moka),,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+2. a) Category,VCP3 Viapath,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+2. b) SubCategory,R81_Congenital myopathy,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+2. c) Panel,VCP3 Viapath R81_Congenital myopathy,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+3a). Is this BED file used for Exome depth?,YES,
+3b). Is this BED file used to generate read counts or filter calls?,filter calls,
+4. a) Is this BED file exclusively based on coordinates provided by a supplier ,No,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+4. b) If yes, does this need further padding?",N/A,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+5. a) Should specific transcripts be used to create the bed file? (as opposed to all transcripts) Specific transcripts - see Moka,Yes - see below,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+5. b) If specific transcripts are to be used have you provided multiple transcripts for the same gene? (if Yes please state),No,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (aledj2) [inline LiveBedfiles/RequestForms/Pan4533.csv:16]: yes

File: LiveBedfiles/Pan4533dataRefSeqFormat.txt [hunk 3]
```diff
@@ -0,0 +1,57 @@
+#bin	name	chrom	strand	txStart	txEnd	cdsStart	cdsEnd	exonCount	exonStarts	exonSEnds	score	name2	cdsStartStat	cdsEndStat	exonFrames
```
Comment (aledj2) [inline LiveBedfiles/Pan4533dataRefSeqFormat.txt:1]: should this file be deleted considering it doesn't have all regions?

File: LiveBedfiles/Pan4566dataRefSeqFormat.txt [hunk 4]
```diff
@@ -0,0 +1,95 @@
+#bin	name	chrom	strand	txStart	txEnd	cdsStart	cdsEnd	exonCount	exonStarts	exonSEnds	score	name2	cdsStartStat	cdsEndStat	exonFrames
```
Comment (aledj2) [inline LiveBedfiles/Pan4566dataRefSeqFormat.txt:1]: should this be deleted as it's incomplete?

**Review by aledj2 — CHANGES_REQUESTED** [review]

**Review by natashapinto — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/moka-guys/mokabed/238)* status: 0 of 14 files reviewed, 2 unresolved discussions (waiting on @aledj2)

___
*[LiveBedfiles/Pan4566dataRefSeqFormat.txt, line 1 at r1](https://reviewable.io/reviews/moka-guys/mokabed/238#-MdsxCOsKvLB9lcJdZqN-r1-1:-Mdw9mny7O9-vGAewVum:b-896fix) ([raw file](https://github.com/moka-guys/mokabed/blob/5da3451f493e1077eea8863ca99ab38186a69400/LiveBedfiles/Pan4566dataRefSeqFormat.txt#L1)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

should this be deleted as it's incomplete?
</blockquote></details>

Done.
___
*[LiveBedfiles/Pan4533dataRefSeqFormat.txt, line 1 at r1](https://reviewable.io/reviews/moka-guys/mokabed/238#-MdsxCOsKvLB9lcJdZqJ-r1-1:-Mdw9o060sthPHsAnA6d:b-896fix) ([raw file](https://github.com/moka-guys/mokabed/blob/5da3451f493e1077eea8863ca99ab38186a69400/LiveBedfiles/Pan4533dataRefSeqFormat.txt#L1)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

should this file be deleted considering it doesn't have all regions?
</blockquote></details>

Done.

**Review by aledj2 — COMMENTED** [review]
> Reviewed 3 of 14 files at r1, 1 of 2 files at r2, 1 of 2 files at r3, 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/moka-guys/mokabed/238)* status: 5 of 14 files reviewed, 1 unresolved discussion (waiting on @aledj2)

File: LiveBedfiles/Pan4566dataRefSeqFormat.txt (hunk 4 above)
Comment (aledj2) [reply LiveBedfiles/Pan4566dataRefSeqFormat.txt:1]: this is still here (I think)

File: LiveBedfiles/RequestForms/Pan4566.csv (hunk 1 above)
Comment (aledj2) [reply LiveBedfiles/RequestForms/Pan4566.csv:16]: not changed

**Review by aledj2 — APPROVED** [review]

**Comment by aledj2** [comment]
This covered Pan4533 and Pan4566

**Pull request merged**